Userland/Libraries/LibCoreDump/Backtrace.cpp

```diff
@@ -95,11 +102,30 @@ String Backtrace::Entry::to_string(bool color) const
         return builder.build();
     }
     builder.appendff("[{}] {}", object_name, function_name.is_empty() ? "???" : function_name);
-    if (source_position.has_value()) {
-        auto& source_position = this->source_position.value();
-        auto fmt = color ? " (\033[34;1m{}\033[0m:{})" : " ({}:{})";
-        builder.appendff(fmt, LexicalPath(source_position.file_path).basename(), source_position.line_number);
+    builder.append(" (");
+
+    Vector<Debug::DebugInfo::SourcePosition> source_positions;
+
+    for (auto& position : source_position_with_inlines.inline_chain) {
+        if (!source_positions.contains_slow(position))
+            source_positions.append(position);
+    }
+
+    if (source_position_with_inlines.source_position.has_value() && !source_positions.contains_slow(source_position_with_inlines.source_position.value())) {
+        source_positions.insert(0, source_position_with_inlines.source_position.value());
+    }
+
+    for (size_t i = 0; i < source_positions.size(); ++i) {
+        auto& position = source_positions[i];
+        auto fmt = color ? "\033[34;1m{}\033[0m:{}" : "{}:{}";
+        builder.appendff(fmt, LexicalPath(position.file_path).basename(), position.line_number);
+        if (i != source_positions.size() - 1) {
+            builder.append(" => ");
+        }
     }
+
+    builder.append(" )");
```

Comment:
```suggestion
    builder.append(")");
```

Is that intentional? :)

Reply:
nope